modules/psap-driver-toolkit-using.adoc

```diff
@@ -80,7 +77,6 @@ spec:
       WORKDIR simple-kmod
 
       # Prep and build the module
-      RUN make buildprep KVER=$(rpm -q --qf "%{VERSION}-%{RELEASE}.%{ARCH}"  kernel-core) KMODVER=${KMODVER} \
       && make all       KVER=$(rpm -q --qf "%{VERSION}-%{RELEASE}.%{ARCH}"  kernel-core) KMODVER=${KMODVER} \
```

Comment:
This line needs to start with RUN. It can't start with "&&". It should be `RUN make all       KVER=$(rpm -q --qf "%{VERSION}-%{RELEASE}.%{ARCH}"  kernel-core) KMODVER=${KMODVER} \`

Reply:
corrected line